### ISSUE TITLE
output images scaled to [-1,1] or [0,1] ?

### ISSUE THREAD

**Issue by starrywintersky** [GaParmar/img2img-turbo#26]
(in your code of class PairedDataset)

# output images scaled to -1,1
      output_t = self.T(output_img)
      output_t = F.to_tensor(output_t)
      output_t = F.normalize(output_t, mean=[0.5], std=[0.5])

**Comment by wujinlonglovezhangmiao1314** [comment]
> Hi,
> 
> The input images are scaled to [0,1] and output images are scaled to [-1,1]. See this code block [here](https://github.com/GaParmar/img2img-turbo/blob/main/src/my_utils/training_utils.py#L290).

hi, i wonder know why  input images are scaled to [0,1] ? why not to [-1,1] as same as antoencoder's input.
